test/sqlancer/mzcompose.py

```diff
@@ -81,18 +81,18 @@ def kill_sqlancer_with_delay() -> None:
         "materialize",
         "--oracle",
         args.oracle,
-        capture=True,
-        capture_stderr=True,
         check=False,
+        detach=True,
+        capture=True,
     )
+    container_id = result.stdout.strip()
 
-    if result.returncode not in (0, 130):
-        raise Exception(
-            f"[SQLancer] Unexpected return code in SQLancer: {result.returncode}\n{result.stdout}\n{result.stderr}"
-        )
+    thread = Thread(target=print_logs, args=(container_id,))
```

Comment:
A comment here about why this is done would be nice. The commit message does not have much information, one needs to drill down all the way to the original ticket to get an explanation.

Reply:
Done